browser/src/control/AutoCompletePopup.ts

```diff
@@ -68,14 +69,14 @@ abstract class AutoCompletePopup {
 	abstract onAdd(): void;
 
 	closePopup(): void {
-		var popupExists = L.DomUtil.get(this.popupId);
+		var popupExists = window.L.DomUtil.get(this.popupId);
 		if (!popupExists) return;
 
 		this.map.jsdialog.focusToLastElement(this.popupId);
 		this.map.jsdialog.clearDialog(this.popupId);
 	}
 
-	abstract getPopupEntries(ev: FireEvent): Array<TreeEntryJSON>;
+	abstract getPopupEntries(ev: FireEvent): any[];
```

Comment:
why? `Array<TreeEntryJSON>` was before

Reply:
When i declare the types more strictly, some type errors occurred. I fixed them i guess. But this one is a real error. The extended class has different return type. Since this aims to have non-functional changes, i didn't touch this one.